src/web-auth/captcha.js

```diff
@@ -391,12 +399,15 @@ function render(auth0Client, element, options, callback) {
  * @param {Function} [options.templates.recaptcha_enterprise] template function receiving the challenge and returning a string
  * @param {Function} [options.templates.hcaptcha] template function receiving the challenge and returning a string
  * @param {Function} [options.templates.friendly_captcha] template function receiving the challenge and returning a string
+ * @param {Function} [options.templates.arkose] template function receiving the challenge and returning a string
  * @param {Function} [options.templates.error] template function returning a custom error message when the challenge could not be fetched, receives the error as first argument
  * @param {String} [options.lang=en] the ISO code of the language for recaptcha
- * @param {Object} [options.callbacks] An optional object containing callbacks called after captcha events (only for Arkose captcha provider)
- * @param {Function} [options.callbacks.onSolved] An optional callback called after the captcha is solved (only for Arkose captcha provider)
- * @param {Function} [options.callbacks.onError] An optional callback called after the captcha encounters an error with the error passed as the first argument (only for Arkose captcha provider)
  * @param {Function} [callback] An optional callback called after captcha is loaded
+ * @returns {Object} An object containing the methods reload, getValue, triggerCaptcha and requiresTrigger.
+ * - reload loads the captcha again
+ * - getValue returns the captcha response
+ * - requiresTrigger after the captcha is loaded, returns true if the provider requires a trigger (Arkose)
+ * - triggerCaptcha after the captcha is loaded, will run the captcha for providers that require a trigger (Arkose)
```

Comment:
Let's remove the added returns documentation here. A return value is not something we want, nor should promote in a callback based API.